repo/compositeArtifacts.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?compositeArtifactRepository version='1.0.0'?>
+<repository name='GWT Eclipse Plugin'
+    type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' 
+    version='1.0.0'>
+  <properties size='1'>
+    <property name='p2.timestamp' value='1665755219001'/>

```

Comment:
What is this timestamp? What's the purpose of it?

Reply:
The timestamp is just something eclipse created. I don't know what's it good for. Plugins are installable even if the timestamp is always the same.